service/exporter/exporter.go

```diff
@@ -57,12 +20,52 @@ type namespace struct {
 	Name string
 	Pods []pod
 }
-type leanixExport struct {
+type Response struct {
 	Namespaces []namespace
 	LastUpdate time.Time
 }
 
-func getNamespaces(c *kubernetes.Clientset) []namespace {
+type Config struct {
+	Excludes []string
+}
+
+// New creates a new configured version service.
+func New(config Config) (*Service, error) {
+	// Settings.
+	if config.Excludes == nil {
+		return nil, microerror.MaskAnyf(invalidConfigError, "config.Excludes must not be empty")
+	}
+	newService := &Service{
+		Config: config,
+	}
+
+	return newService, nil
+}
+
+// Service implements the version service interface.
+type Service struct {
+	Config
+}
+
+func (s *Service) Get(ctx context.Context) (*Response, error) {
+
+	config, err := rest.InClusterConfig()
+	if err != nil {
+		return nil, err
```

Comment:
Regarding this, in the docs, it specify to used typed error. In this case, I have to declare new errors encapsulating the one generated by `rest.InClusterConfig()` or just return `return nil, microerror.MaskAny(err)' ?